src/docs/cookbook/design/package-fonts.md

```diff
@@ -12,7 +12,7 @@ Rather than declaring a font as part of an app,
 you can declare a font as part of a separate package.
 This is a convenient way to share the same font across
 several different projects,
-or for coders publishing their packages to the [Pub site][].
+or for coders publishing their packages to the [pub.dev site][].
```

Comment:
GLOBAL: The links are inconsistent. I'd use either "to [pub.dev][]" or "to the [pub.dev site][]".